library/junos_install_os

```diff
@@ -265,23 +294,17 @@ def main():
     try:
         from jnpr.junos import Device
         from jnpr.junos.version import VERSION
-        if not LooseVersion(VERSION) >= LooseVersion('1.2.2'):
-            module.fail_json(msg='junos-eznc >= 1.2.2 is required for this module')
+        if not LooseVersion(VERSION) >= LooseVersion('2.1.2'):
+            module.fail_json(msg='junos-eznc >= 2.1.2 is required for this '
+                                 'module')
     except ImportError as ex:
         module.fail_json(msg='ImportError: %s' % ex.message)
 
-    if int(args['port']) == 23 and LooseVersion(VERSION) < LooseVersion('2.0.0'):
-        module.fail_json(msg='junos-eznc >= 2.0.0 is required for telnet connection.')
-
-    if (module.boolean(args['issu']) or module.boolean(args['nssu'])) and \
```

Comment:
Have you done the changes on top of latest code. Why issu/nssu check has been removed?